### PR TITLE
58 standardize logging

### PR DESCRIPTION
**Pull request by ckulfan**
Created a logger class to wrap Winston logging, which I added to the project. Then updated all instances of logging to use the wrapper. Finally, included unit tests for the logger.

### CODE REVIEW

File: src/Logger.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/**
+ * Logger.js
+ * ============
+ * Wrapper around Winston logger capability that will allowus to standardize logging for FTServer more easily.
+ *
+ * For now I am keeping the console logger with default level of info. The log level can be overridden via an environment varaible.
+ * I setup cli logging to keep things consistent with the way we had before, but we could switch to json logging and add a timestamp
+ * by adding the following
```
Comment (alexmarc-us) [inline src/Logger.js:8]: Is there a missing comment here? What is "the following"?

File: src/Logger.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  warn(message) {
+    this.logger.log('warn', message);
+  }
+
+  error(message) {
+    this.logger.log('error', message);
+  }
+}
+
+export default Logger;
```
Comment (alexmarc-us) [inline src/Logger.js:39]: Nit: Newline at the end of the file.

File: src/Logger.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+/**
+ * Logger.js
+ * ============
+ * Wrapper around Winston logger capability that will allowus to standardize logging for FTServer more easily.
+ *
+ * For now I am keeping the console logger with default level of info. The log level can be overridden via an environment varaible.
```
Comment (alexmarc-us) [inline src/Logger.js:6]:
```suggestion
 * For now I am keeping the console logger with default level of info. The log level can be overridden via an environment variable.
```

**Review by alexmarc-us — CHANGES_REQUESTED** [review]
Overall this looks great. Just a couple of nit picking requests.

File: src/Logger.js (hunk 1 above)
Comment (ckulfan) [reply src/Logger.js:8]: Fixed the comment to refer to the guide

File: src/Logger.js (hunk 2 above)
Comment (ckulfan) [reply src/Logger.js:39]: added newline

File: src/Logger.js (hunk 3 above)
Comment (ckulfan) [reply src/Logger.js:6]: fixed the typo with the edits above.

**Review by alexmarc-us — APPROVED** [review]

**Pull request merged**